### PR TITLE
process: Added issue templates- Code-Health, Research, Process

### PR DESCRIPTION
**Pull request by Ana2k**
fixes: #350 

Chore.md file has been added inside issue template to show chore option while creating issues.

### CODE REVIEW

**Comment by jingtang10** [comment]
@Ana2k thanks for your PR! In the future please discuss with admins and code owners in the issue before raising PRs like this that change the processes.

@mberg @fredhersch @pld fyi.

**Comment by Ana2k** [comment]
> @Ana2k thanks for your PR! In the future please discuss with admins and code owners in the issue before raising PRs like this that change the processes.
> 
> @mberg @fredhersch @pld fyi.

@jingtang10 thanks for the heads up. I will keep this in mind from now on.

**Comment by Ana2k** [comment]
@jingtang10 I have implemented the changes on this as requested. 
Please review it.

File: .github/ISSUE_TEMPLATE/code_health.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name:  Code-Health
+about: Issues related to Refactoring, Improving Testing, Infrastructure, Documentation or related.
```
Comment (jingtang10) [inline .github/ISSUE_TEMPLATE/code_health.md:3]:
```suggestion
about: Code refactoring, testing infrastructure, documentation, etc.
```

File: .github/ISSUE_TEMPLATE/code_health.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+name:  Code-Health
```
Comment (jingtang10) [inline .github/ISSUE_TEMPLATE/code_health.md:2]:
```suggestion
name:  Code health
```

File: .github/ISSUE_TEMPLATE/process.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+name:  Process
+about: Issues related to builds and releases, communications, templates etc.
```
Comment (jingtang10) [inline .github/ISSUE_TEMPLATE/process.md:3]:
```suggestion
about: Builds, releases, communications, etc.
```

File: .github/ISSUE_TEMPLATE/research.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+---
+name:  Research
+about: Issues related to creation and managing of research tasks. Includes background, goal, and methodology.
```
Comment (jingtang10) [inline .github/ISSUE_TEMPLATE/research.md:3]:
```suggestion
about: Research tasks
```

File: .github/ISSUE_TEMPLATE/research.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+---
+
+**Describe the Issue**
+
+<!--Please state here specifics of what you want to do.-->
```
Comment (jingtang10) [inline .github/ISSUE_TEMPLATE/research.md:9]:
```
**Describe the issue to be researched**
Include any background information and available resources.

**Describe the goal of the research**
What's the desired outcome of this task? What artifacts should be produced?

**Describe the methodology**
What information is available for the research? Where can more information be found? Who should the assignee approach to ask questions?
```

File: .github/ISSUE_TEMPLATE/code_health.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+---
+
+**Describe the Issue**
+
+<!--Please state here specifics of what you want to do.-->
+
+**Would you like to work on the issue?**
+
+<!--Please state if this issue should be assigned to you or who you think could help to solve this issue.-->
```
Comment (jingtang10) [inline .github/ISSUE_TEMPLATE/code_health.md:13]:
```suggestion
**Describe the issue**

**Would you like to work on the issue?**

```

and below for the other two templates.

File: .github/ISSUE_TEMPLATE/code_health.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
+---
+name:  Code-Health
+about: Issues related to Refactoring, Improving Testing, Infrastructure, Documentation or related. 
+
```
Comment (jingtang10) [inline .github/ISSUE_TEMPLATE/code_health.md:4]: add a row
```
labels: 'code health'
```

File: .github/ISSUE_TEMPLATE/process.md [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+
```
Comment (jingtang10) [inline .github/ISSUE_TEMPLATE/process.md:4]: add a row
```
labels: 'process'
```

File: .github/ISSUE_TEMPLATE/research.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
+---
+name:  Research
+about: Issues related to creation and managing of research tasks. Includes background, goal, and methodology.
+
```
Comment (jingtang10) [inline .github/ISSUE_TEMPLATE/research.md:4]: add a row
```
labels: research
```

**Review by jingtang10 — CHANGES_REQUESTED** [review]
While you're at it, can you add the `labels` field to the two existing templates too? They shoould be `bug` and `enhancement`. Thanks!

**Comment by Ana2k** [comment]
@jingtang10 i have implemented the changes. Please review them. Thanks!

File: .github/ISSUE_TEMPLATE/research.md [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+**Describe the issue to be researched**
+Include any background information and available resources.
+
+**Describe the goal of the research**
+What's the desired outcome of this task? What artifacts should be produced?
+
+**Describe the methodology**
+What information is available for the research? Where can more information be found? Who should the assignee approach to ask questions?
```
Comment (jingtang10) [inline .github/ISSUE_TEMPLATE/research.md:17]: tweak this a little:

```suggestion
Where can more information be found? Who should the assignee approach to ask questions? How can a decision be made?
```

thanks!

**Review by jingtang10 — APPROVED** [review]
Thanks! 👍 
just one little nit.

**Pull request merged**